myhpi/settings.py

```diff
@@ -196,7 +196,7 @@
 
 # Base URL to use when referring to full URLs within the Wagtail admin backend -
 # e.g. in notification emails. Don't include '/admin' or a trailing slash
-BASE_URL = "http://example.com"
```

Comment:
maybe we want to change it away from example.com at this point? ^^

Reply:
what would you suggest?

Reply:
either write https://myhpi.de here or move it to an environment variable

Reply:
Can't we just use the SITE_URL env variable?

Reply:
sounds good

Reply:
yes!